integrationtests/tests/tests.rs

```diff
@@ -141,6 +143,65 @@ async fn wallet_peg_outs_are_only_allowed_once_per_epoch() -> Result<()> {
     .await
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn wallet_peg_outs_support_rbf() -> Result<()> {
+    test(2, |fed, user, bitcoin, _, _| async move {
+        // Need lock to keep tx in mempool from getting mined
+        let bitcoin = bitcoin.lock_exclusive().await;
+        let address = bitcoin.get_new_address().await;
+
+        fed.mine_and_mint(&user, &*bitcoin, sats(5000)).await;
+        let (fees, out_point) = user.peg_out(1000, &address).await;
+        fed.run_consensus_epochs(2).await;
+        fed.broadcast_transactions().await;
+
+        let txid = user
+            .client
+            .wallet_client()
+            .await_peg_out_outcome(out_point)
+            .await
+            .unwrap();
+        assert_eq!(
+            bitcoin.get_mempool_tx_fee(&txid).await,
+            fees.amount().into()
+        );
+
+        // RBF by increasing sats per kvb by 1000
+        let rbf = Rbf {
+            fees: PegOutFees {
+                fee_rate: Feerate { sats_per_kvb: 1000 },
+                total_weight: fees.total_weight,
+            },
+            txid,
+        };
+        let out_point = user.client.rbf_tx(rbf.clone()).await.unwrap();
+        fed.run_consensus_epochs(2).await;
+        fed.broadcast_transactions().await;
+        let txid = user
+            .client
+            .wallet_client()
+            .await_peg_out_outcome(out_point)
+            .await
+            .unwrap();
+
+        assert_eq!(
+            bitcoin.get_mempool_tx_fee(&txid).await,
+            (fees.amount() + rbf.fees.amount()).into()
+        );
+
+        assert_eq!(
+            bitcoin.mine_block_and_get_received(&address).await,
+            sats(1000)
+        );
+        bitcoin
+            .mine_blocks(fed.wallet.consensus.finality_delay as u64)
+            .await;
+        fed.run_consensus_epochs(1).await;
+        assert_eq!(fed.max_balance_sheet(), 0);
```

Comment:
Why this check? Does the test assume it is sweeping all the funds from the federation?

Reply:
Good question!  If the federation doesn't set fees, it should never be at a profit.

This check verifies RBF `PendingTransactions` are counted as liabilities rather than assets (in the normal case `PendingTransactions` represent a change UTXO asset we will receive)